src/notification.js

```diff
@@ -34,9 +35,9 @@ export class Notification {
     this.setContainer();
     let aureliaComposedListener = () => {
       this.setContainer();
-      document.removeEventListener('aurelia-composed', aureliaComposedListener);
+      DOM.removeEventListener('aurelia-composed', aureliaComposedListener);
```

Comment:
Ah yeah, just that DI allows you to even overwrite DOM with a different implementation. That's why DI is cool. But I do understand your reasoning, so I'll :+1: